scanapi/evaluators/code_evaluator.py

```diff
@@ -19,6 +19,23 @@ class CodeEvaluator:
 
     @classmethod
     def evaluate(cls, sequence, spec_vars, is_a_test_case=False):
+        """Evaluate a sequence of python code and return a evaluation indicator.
+
+        Args:
+            sequence [string]: string representing python statements to be
```

Comment:
Can you please update this Inconsistent spacing: `sequence [string]` -> `sequence[string]` and `spec_vars [dict]` -> `spec_vars[dict]`